nasa-gibs/worldview
# WV-2629 date intervals

## Description
Layers like GEDI's Aboveground Biomass, Canopy Height Standard Deviation Canopy Height Mean show the incorrect Temporal Coverage in the layer info box. This also causes imagery to not load for the effected layers.

## How To Reproduce
- Open production WV
- Load layer GEDI Aboveground biomass
- Click the info button for the layer; note the Temporal Coverage is 2019 Apr 18 - 2019 Apr 18
- Try to load imagery on 2019 Apr 18 & confirm no imagery displays

## How To Test
- git pull wv-2629-date-intervals
- npm run build
- npm run watch
- Load layer GEDI Aboveground biomass
- Click the info button for the layer; note the Temporal Coverage is 2019 Apr 18 - 2019 Apr 18
- Try to load imagery on 2019 Apr 18 & confirm no imagery displays

@nasa-gibs/worldview
